vue-client/src/components/search/facet-group.vue

```diff
@@ -237,13 +268,20 @@ export default {
             :show-iconless="false"
             v-if="group.dimension === 'instanceOf.@type' || group.dimension === '@type'"
             :type="facetItem.object['@id']" />
+          <check-box v-if="group.dimension === 'concerning.@reverse.itemOf.heldBy.@id' && isChangeView"
```

Comment:
This check should be on the presence of the `_selected` property so that it works for any multi-select facet group.